design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -190,6 +208,7 @@ type NegotiableQuote {
     created_at: String @doc(description: "Timestamp indicating when the negotiable quote was created.")
     updated_at: String @doc(description: "Timestamp indicating when the negotiable quote was updated.")
     status: NegotiableQuoteStatus!
+    shipping_addresses: [ShippingCartAddress]!
```

Comment:
Need help deciding what to do here. The structure for ShippingCartAddress in the QuoteGraphQL module is as follows. Do we need to duplicate _each_ of them for NegotiableQuote?

` CartAddressInterface` -> `ShippingCartAddress` -> `CartAddressRegion`/`CartAddressCountry`

Reply:
Yes